flow/client_server/templates/navbar/base.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
```

Comment:
Does this need to say !DOCTYPE html for it to use html at all because on some it doesn't say this.